eng/pipelines/eng-common-sync.yml

```diff
@@ -0,0 +1,36 @@
+# Mirror the eng/common folder to all subscribed langauge repos.
+
+trigger:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - eng/common
+
+pr: none
+
+resources:
+  repositories:
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
+
+pool:
+  vmImage: windows-2019
+
+jobs:
+  - job: SyncEngCommon
+    displayName: Sync eng/common Directory
+
+    steps:
+    - template: tools/clone-buildtools/clone-buildtools.yml@azure-sdk-build-tools
+    - template: ./templates/steps/sync-directory.yml
+      parameters:
+        CommitMessage: Sync eng/common directory with azure-sdk-tools repository
+        DirectoryToSync: eng/common
+        Repos:
+          - azure-sdk-for-java
+          - azure-sdk-for-js
+          - azure-sdk-for-net
+          - azure-sdk-for-python
```

Comment:
Will need to add ios and android after they merge with master. In the meantime I'll manually sync (I think an elaborate multi-branch merging model would be unnecessary - and just be a conflict magnet). What about azure-sdk-for-c and azure-sdk-for-go?

Reply:
I think we can go ahead and add all the other repo's we want right now. The worst case they will have files they don't yet use.

Reply:
I added all the other repo's and ensure the azure-sdk account had a fork for them.